styles.css/main.css

```diff
@@ -119,6 +132,7 @@ nav ul li {
 }
 
 .favorite {
+  /* See the sample solution for one approach */
   display: inline-flexbox;
```

Comment:
It looks like you didn't incorporate the changes in the favourites tab from the last lab.  I didn't have time to work everything out on your site for this, so please refer to the sample solution in the LMS to see one approach to the problem.